metrotranscom/doorway
# fix: remove view listing button from closed listings

This PR addresses #704


## Description

Hides "view application" button on `/account/applications` when listing is closed.

## How Can This Be Tested/Reviewed?

File application, it should show `view application` button on `/account/applications`. Close listing, the button should be hidden.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
